packages/apps/composer-app/src/internal.ts

```diff
@@ -0,0 +1,14 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { storeObservabilityGroup } from '@dxos/observability';
+
+import { appKey } from './constants';
+
+const run = async () => {
+  await storeObservabilityGroup(appKey, 'dxos');
```

Comment:
could this also set and store the mode to full? or should we just do away with the concept of that and check for the group?

Reply:
At present, I've not seen any difference between basic/full in the browser, the differences seemed more specific to node right now.

Reply:
per the latter comment, i've found it useful to have them separate for testing, but it might depend on how much we want to distinguish our "prod" internal usage with the "development". or how much appetite we have for cleaning data post-ingest

Reply:
There's just a small amount of changes on the metrics side, I didn't understand the intent of the semantics. But adding the device and identity tags is what I had considered gating behind mode === 'full', and could consider what other stuff we have hardcoded to check for the group?

Reply:
At this stage we're going to be relying to the truncated identity key for user identification, so I think that falls into `basic` for now. Certainly once we're at this stage https://github.com/orgs/dxos/discussions/5438 then it would be nice to have a way to still include it optionally for debugging purposes and that sounds like `full` mode?

Reply:
ok, that makes sense. will remove the check and start documenting the behavior in the README.